homeassistant/components/pushover/notify.py

```diff
@@ -47,12 +48,41 @@ def __init__(self, user_key, api_token):
     def send_message(self, message='', **kwargs):
         """Send a message to a user."""
         from pushover import RequestError
+        import re
```

Comment:
imports are on top but can be removed with latest comment before

Reply:
Changes made.